### PR TITLE
Add `removeExtensionUsed` and `removeExtensionRequired`

### PR DESCRIPTION
**Pull request by lilleyse**
Adds helper functions for removing extensions from `extensionsUsed` and `extensionsRequired` and fixes a bug where `KHR_texture_transform` wasn't removed in `applyKhrTextureTransform`.

### CODE REVIEW

File: CesiumGltfReader/src/applyKhrTextureTransform.cpp [hunk 1]
```diff
@@ -118,5 +118,7 @@ void applyKhrTextureTransform(Model& model) {
       }
     }
   }
+
```
Comment (j9liu) [inline CesiumGltfReader/src/applyKhrTextureTransform.cpp:121]: Remove extra space?

**Review by j9liu — COMMENTED** [review]
Looks good @lilleyse ! Noticed one extra line break, but I'll just push a fix and merge after.

File: CesiumGltfReader/src/applyKhrTextureTransform.cpp (hunk 1 above)
Comment (lilleyse) [reply CesiumGltfReader/src/applyKhrTextureTransform.cpp:121]: I think it's ok to keep the extra space, it's consistent with the the files below

Comment (j9liu) [reply CesiumGltfReader/src/applyKhrTextureTransform.cpp:121]: Sorry, I had split view enabled on Github so it looked like there were two line breaks in a row. 😬 This is fine!

**Pull request merged**